scripts/trivia.js

```diff
@@ -2443,8 +2444,23 @@ addUserCommand(["triviarules"], function (src, commandData, channel) {
     sys.sendMessage(src, "- Trivia is a game for all to enjoy, and bad sportsmanship can turn that into a less pleasant experience for everyone involved. Do not brag when you get a question right, and do not be a sore loser when you get it wrong. Any form of cheating isn't allowed, whether by gaining an unfair advantage for yourself, or hampering someone else's ability to answer.", channel);
     sys.sendMessage(src, "4. Do not abuse question submission:", channel);
     sys.sendMessage(src, "- This includes, but is not limited to, submitting questions that are plainly wrong, offensive to other users, racist, inappropriate, trolling, spamming the same question multiple times, purposefully submitting questions that are already in the database. Trivia Admins can revoke a user's submitting privileges at any time if they deem it necessary.", channel);
+    sys.sendMessage(src, "5. Trivia House Rules apply to all Trivia games:", channel);
+    sys.sendHtmlMessage(src, "<timestamp/> - Use <b><a href=\"po:send//houserules\">/houserules</a></b> to view the Trivia House Rules.", channel);
 }, "Shows the rules of the Trivia channel.");
 
+addUserCommand(["houserules"], function (src, commandData, channel) {
```

Comment:
Maybe a note on how questions pertaining to base stats account for Mega forms, since I notice that answers that are Megas tend to take some users by surprise.